src/Azure/Orleans.Statistics.AzureStorage/Storage/ClientMetricsTableDataManager.cs

```diff
@@ -66,21 +67,22 @@ internal class ClientMetricsTableDataManager : IClientMetricsDataPublisher
         private readonly ILogger logger;
         private readonly ILoggerFactory loggerFactory;
         private static readonly TimeSpan initTimeout = AzureTableDefaultPolicies.TableCreationTimeout;
-
-        public ClientMetricsTableDataManager(ILoggerFactory loggerFactory, IOptions<ClusterClientOptions> clusterClientOptions)
+        private StorageOptions storageOptions;
```

Comment:
Storage options have only been used on the cluster side previously.  I don't think they are being configured from the client config on the client side.